src/content/language/built-in-types.md

```diff
@@ -173,6 +173,18 @@ const msUntilRetry = secondsUntilRetry * msPerSecond;
 
 For more information, see [Numbers in Dart][dart-numbers].
 
+You can use one or more underscores (`_`) as digit separators
+to make long number literals more readable.
+Multiple digit separators allow for higher level grouping.
+
+<?code-excerpt "misc/lib/language_tour/built_in_types.dart (digit-separators)"?>
+```dart
+var n1 = 1_000_000;
+var n2 = 0.000_000_000_01;
+var n3 = 0x00_14_22_01_23_45;  // MAC address
+var n4 = 555_123_4567;  // US Phone number
+var n5 = 100__000_000__000_000;  // one hundred million million!
+```
```

Comment:
Consider adding a version note about the required language version.

Reply:
Ah yeah this only works if your pubspec specifies `sdk: ^3.6.0` or similar.